### PR TITLE
fix: remove jobIds from query

### PR DESCRIPTION
**Pull request by eisbilir**
After header links for pagination added it returns 502 error if jobIds array has more than ~40 ids.
Especially when all of the first,next and last links are exist.
Postman 2021-06-06 19 59 05

when we make page=2, the prev-page link added to the header

Postman 2021-06-06 19 56 34

### CODE REVIEW

File: src/controllers/JobController.js [hunk 1]
```diff
@@ -61,6 +61,9 @@ async function searchJobs (req, res) {
     req.query.jobIds = req.body.jobIds
   }
   const result = await service.searchJobs(req.authUser, req.query)
+  if (req.query.jobIds) {
+    delete req.query.jobIds
+  }
```
Comment (maxceem) [inline src/controllers/JobController.js:66]: The current logic to add and then to delete fields is a bit hard to follow. It would be nice if logic would more straightforward, for example like this:

```js
const query = {
   ...req.query,
   // we can pass jobIds in `body` or in `query`
   jobIds: _.get(req, 'body.jobIds', req.query.jobIds)
}
const result = await service.searchJobs(req.authUser, query)

Comment (LieutenantRoger) [reply src/controllers/JobController.js:66]: I agree, maybe simply check body for it, if jobIds are not present, we can avoid to pass jobIds to the criteria or  pass an empty array.  By looking at the service method, seems both would be fine.

**Comment by eisbilir** [comment]
fixed by #315
